internal/util/util.go

```diff
@@ -75,6 +75,8 @@ var (
 	GitCommit string
 	// DriverVersion which will be driver version
 	DriverVersion string
+	// LogMessage specifies the log message to display
+	LogMessage string
```

Comment:
Where is this used?

Reply:
Oh my bad, added it for testing. Will get rid of it.